packages/api/src/routes/medical/internal-docs.ts

```diff
@@ -311,7 +312,7 @@ router.post(
  * @param req.query.cxId - The customer/account's ID.
  * @param req.query.patientId - The customer/account's ID.
  * @param req.query.facilityId - Optional; The facility providing NPI for the document query.
- *
+ * @param req.body Optional metadata to be sent through Webhook. WH disable can be sent here.
```

Comment:
> WH disable can be sent here.

It would be better to use the exact property name, "WH disable" doesn't tell me what I should send on the payload.